libs/ledger-live-common/src/modularDrawer/hooks/useRightBalanceAsset.tsx

```diff
@@ -0,0 +1,154 @@
+import { useMemo } from "react";
+import type { ReactNode } from "react";
+import type { CryptoOrTokenCurrency, Currency } from "@ledgerhq/types-cryptoassets";
+import type { CurrenciesByProviderId } from "@ledgerhq/live-common/deposit/type";
```

Comment:
circular deps will happen use relative path instead for every `@ledgerhq/live-common/`